pkgs/tools/inputmethods/fcitx5/fcitx5-bamboo.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, fcitx5
+, gettext
+, go
+}:
+stdenv.mkDerivation rec {
+  pname = "fcitx5-bamboo";
+  version = "1.0.4";
+
+  src = fetchFromGitHub {
+    owner = "fcitx";
+    repo = "fcitx5-bamboo";
+    rev = version;
+    hash = "sha256-EcpuZN2JU6HSuiQgBPBsoYftdHypiyFlrUxDBlVW6eo=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    gettext
+    go
+  ];
+
+  buildInputs = [
+    extra-cmake-modules
+    fcitx5
+  ];
+
+  env.GOCACHE = "/tmp/go-cache";
+
+  meta = {
+    description = "Vietnamese input method engine support for Fcitx";
+    homepage = "https://github.com/fcitx/fcitx5-bamboo";
+    license = lib.licenses.lgpl21Plus;
+    maintainers = with lib.maintainers; [ eclairevoyant ];
+    platforms = lib.platforms.all;
```

https://github.com/NixOS/nixpkgs/blob/fa804edfb7869c9fb230e174182a8a1a7e512c40/pkgs/tools/inputmethods/fcitx5/default.nix#L106

```suggestion
    platforms = lib.platforms.linux;
```

---

Thanks, will squash afterwards.